src/app/page-meta.service.ts

```diff
@@ -20,7 +21,7 @@ export class PageMetaService {
     private title: Title,
   ) {}
 
-  setCommon(title: string, description: string, imageUri: string | null) {
+  setCommon(title: string, description: string = '', imageUri: string | null = null) {
```

Comment:
IMHO meta descriptions have very limited value, and none when set client side. Not worth requiring ourselves to type in a description every time.